pywbemtools/pywbemcli/_pywbem_server.py

```diff
@@ -436,17 +435,12 @@ def create_connection(self, log=None, use_pull=None, pull_max_cnt=None,
                                           self._wbem_server,
                                           self._mock_server,
                                           verbose)
-                # All errors in building mock repository cause abort
-                # Errors where information display handled by build_repository
-                except MOFParseError as pe:
-                    click.echo('Mock repository build exception:\n'
-                               '{0}'.format(pe), err=True)
-                    raise click.Abort()
-
-                except Exception as ex:
-                    click.echo('Mock repository build exception. %s.: %s' % (
-                        ex.__class__.__name__, ex), err=True)
-                    raise click.Abort()
+                except click.Abort:
```

Comment:
Why accept and then reraise click.Abort()?

Reply:
In order to exclude click.Abort from the handling of Exception further down, which does not abort.

Reply:
Good